Makefile

```diff
@@ -6,14 +6,21 @@ clean:
 	rm -f preview-extensions-ggqizro707
 	rm -f extensions.zip
 
-dist: clean
+dist-x86_64: clean
 	env GOARCH=amd64 GOOS=linux go build -ldflags="-s -w" -o ./extensions/newrelic-lambda-extension
 	touch preview-extensions-ggqizro707
 
-zip: dist
+dist-arm64: clean
+	env GOARCH=arm64 GOOS=linux go build -ldflags="-s -w" -o ./extensions/newrelic-lambda-extension
+	touch preview-extensions-ggqizro707
+
+zip-x86_64: dist-x86_64
+	zip -r extensions.zip preview-extensions-ggqizro707 extensions
+
+zip-arm64: dist-arm64
 	zip -r extensions.zip preview-extensions-ggqizro707 extensions
```

Comment:
Did we want to give a different name to this output file to indicate the different architecture, or is that not needed?

Reply:
It's not used currently by the CI/CD, but not a bad idea.